client/src/App.css

```diff
@@ -1,3 +1,21 @@
 body {
     margin: 0;
+}
+
+@keyframes rotate-triangle {
```

Comment:
This should not be globally scoped here if you're using it in the loader that is nested 3 files deep in a jsx. I would just include this code there. You generally just want global resets to be here (not always but a good rule of thumb when trying to look for why the animation is acting the way it is for some one else later, they won't know to look here for it).